proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -479,6 +479,30 @@ public void defineWorktable(AST defAST, AST idAST) {
   }
 
   @Override
+  public void widattr(AST widAST, AST idAST, ContextQualifier cq) throws TreeParserException {
+    LOG.trace("Accessing " + idAST.getText() + " in mode " + cq);
+    if (idAST.getType() == NodeTypes.THISOBJECT) {
+      AST tok = idAST.getNextSibling();
+      if (tok.getType() == NodeTypes.OBJCOLON) {
+        AST fld = tok.getNextSibling();
+        String name = fld.getText();
+
+        FieldLookupResult result =  currentBlock.lookupField(name, true);
+        // TODO Once we've added static member resolution, we can re-add this test.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1135)